mcs/class/Mono.Debugger.Soft/Test/dtest.cs

```diff
@@ -71,8 +71,8 @@ void StackTraceDump (Event e)
 #if !MONODROID_TOOLS
 	Diag.ProcessStartInfo CreateStartInfo (string app, string method = null, string runtimeParameters = null) {
 		var pi = new Diag.ProcessStartInfo ();
-		pi.RedirectStandardOutput = true;
-		pi.RedirectStandardError = true;
+		//pi.RedirectStandardOutput = true;
+		//pi.RedirectStandardError = true;
```

Comment:
This is unrelated?

Reply:
It was. Although those lines makes it really hard to debug problems since they had output from the runtime.